webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.js

```diff
@@ -22,9 +24,13 @@ export default class PasswordStrength extends React.Component {
         ? userInputIds.map(input => document.getElementById(input).value)
         : [];
 
+    const showErr = (present, err) => (
+      !present && err ? err : false
```

Comment:
does `showErr` suppose to return boolean or error object?
It's not clear to me.
maybe you can do
```js
const showErr = (present, err) => { !present && err ? err : undefined }
```

Reply:
```js
const showErr = (present, err) =>  !present && err;
```
I believe this is good enough? Although it will return `false`.

Reply:
@sharvit this one is also OK.